src/CUDA_Runtime_Discovery.jl

```diff
@@ -413,6 +418,9 @@ function find_libcudadevrt(toolkit_dirs)
             if Sys.WORD_SIZE == 64
                 push!(all_locations, joinpath(location, "lib64"))
             end
+            if Sys.islinux()
+                push!(all_locations, joinpath(location, "targets", "$(Sys.ARCH)-linux", "lib")) # NVHPC SDK
```

Comment:
On PPC:

```
julia> Sys.ARCH
:powerpc64le
```

So I think you have to do the song and dance of:

```
if Sys.ARCH == :ppc64le || Sys.ARCH == :powerpc64le
```

+ the mapping from aarch64 to `sbsa`